clusters/migrating-to-gcp.md

```diff
@@ -28,7 +28,8 @@ The documentation will be updated when Tokendings is publicly available.
 {% endhint %}
 
 ### Deploy
-Same mechanism as for on-premise clusters. See [GCP clusters][GCP].
+The same deployment mechanism is leveraged for both on-premise K8s clusters and the others non-on-premise.
```

Comment:
Isn't is better to say GCP instead of non-on-premise? Reads easier.

Reply:
Fix'd, thanks! =)